test/conformance/helpers/broker_tracing_test_helper.go

```diff
@@ -45,44 +44,33 @@ func BrokerTracingTestHelperWithChannelTestRunner(
 	setupClient lib.SetupClientOption,
 ) {
 	channelTestRunner.RunTests(t, lib.FeatureBasic, func(t *testing.T, channel metav1.TypeMeta) {
-		BrokerTracingTestHelper(t, brokerClass, channel, setupClient)
+		tracingTest(t, setupClient, setupBrokerTracing(brokerClass), channel)
 	})
 }
 
-// BrokerTracingTestHelper runs the Broker tracing test using the given TypeMeta.
-func BrokerTracingTestHelper(t *testing.T, brokerClass string, channel metav1.TypeMeta, setupClient lib.SetupClientOption) {
-	testCases := map[string]TracingTestCase{
-		"includes incoming trace id": {
-			IncomingTraceId: true,
-		},
-	}
-
-	for n, tc := range testCases {
-		t.Run(n, func(t *testing.T) {
-			tracingTest(t, setupClient, setupBrokerTracing(brokerClass), channel, tc)
-		})
-	}
-}
-
 // setupBrokerTracing is the general setup for TestBrokerTracing. It creates the following:
 // 1. Broker.
 // 2. Trigger on 'foo' events -> K8s Service -> transformer Pod (which replies with a 'bar' event).
 // 3. Trigger on 'bar' events -> K8s Service -> eventdetails Pod.
 // 4. Sender Pod which sends a 'foo' event.
 // It returns a string that is expected to be sent by the SendEvents Pod and should be present in
 // the LogEvents Pod logs.
-func setupBrokerTracing(brokerClass string) SetupInfrastructureFunc {
+func setupBrokerTracing(brokerClass string) SetupTracingTestInfrastructureFunc {
 	const (
-		etTransformer  = "transformer"
-		etLogger       = "logger"
-		defaultCMPName = "eventing"
+		etTransformer = "transformer"
+		etLogger      = "logger"
+		senderName    = "sender"
+	)
+	var (
+		eventID   = uuid.New().String()
```

Comment:
This could be initialized to a constant, randomizing it doesn't provide much value.